cmd/network.go

```diff
@@ -392,199 +335,57 @@ func (c *networkCmd) updateRequestFromCLI(args []string) (*models.V1NetworkUpdat
 		return nil, err
 	}
 
-	c.log.Infow("network command updateRequestFromCLI", "current network", resp)
-
-	addPrefixes := viper.GetStringSlice("add-prefixes")
-	removePrefixes := viper.GetStringSlice("remove-prefixes")
-	addDestinationprefixes := viper.GetStringSlice("add-destinationprefixes")
-	removeDestinationprefixes := viper.GetStringSlice("remove-destinationprefixes")
-
-	c.log.Infow("network command updateRequestFromCLI, update requests:",
-		"add-prefixes", addPrefixes,
-		"remove-prefixes", removePrefixes,
-		"add-destinationprefixes", addDestinationprefixes,
-		"remove-destinationprefixes", removeDestinationprefixes,
+	var (
+		ur = &models.V1NetworkUpdateRequest{
```

Comment:
Tried to set values in the request only if something really changes, but it produced quite a lot of code. Not sure if it was worth it. :/